netsocadmin/register_tools.py

```diff
@@ -23,6 +23,36 @@
 ldap_server = ldap3.Server(config.LDAP_HOST, get_info=ldap3.ALL)
 
 
+def update_password(user: str, password: str):
```

Comment:
Could you change [this](https://github.com/UCCNetsoc/netsocadmin2/pull/74/files?utf8=%E2%9C%93&diff=split&w=1#diff-72835049171ebe8f6353d7064e9126cbR338) to use this function too? :)